src/lib/components/ReactSearchKit/ReactSearchKit.test.js

```diff
@@ -124,15 +141,20 @@ describe('test ReactSearchKit component', () => {
     );
   });
 
-  it('should use inject defaultSortByOnEmptyQuery in the configuration', () => {
+  it('should use layout options in the initialQueryState', () => {
     shallow(
-      <ReactSearchKit searchApi={searchApi} defaultSortByOnEmptyQuery="value" />
+      <ReactSearchKit
```

Comment:
nice thanks a lot!!!
Can you also add a test if you pass something else that is not list/grid?
And if you pass an invalid key?